internal/petdemo/handlers/get_random_petpic_read.go

```diff
@@ -0,0 +1,46 @@
+package handlers
+
+import (
+	"context"
+
+	"github.com/anz-bank/sysl-template/internal/gen/flickr"
+	"github.com/anz-bank/sysl-template/internal/gen/petdemo"
+	"github.com/anz-bank/sysl-template/internal/gen/petstore"
+)
+
+/* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ * get random pet pic
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
```

Comment:
Nit: this comment is both redundant and strangely formatted. Prefer plain `// Thing does foo` comments on each exported name (https://golang.org/doc/effective_go.html#commentary).

Reply:
Corrected